test/cereal_test.exs

```diff
@@ -2,7 +2,43 @@ defmodule CerealTest do
   use ExUnit.Case
   doctest Cereal
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  defmodule UserSerializer do
+    use Cereal.Serializer
+    attributes [:name, :_context]
```

Comment:
Requiring all context-needing serializers to define the `:_context` attribute feels pretty heavy-handed, especially since it's stylized as an internal concern. Would it be possible to add the `:_context` attribute to every attributes list via an update to the `attributes/1` marco itself?

Reply:
I updated the test to make it a bit more clear (maybe). Basically the idea is not for everything we serialize to actually return the `_context` in its own attributes, but rather to be able to use it as a part of its normal serialization.

the use case here is that when I load an asset from a presentation, for example, I want the asset serializer to be able to refer to that presentation so it can format the hls token accordingly.